api/v1alpha1/sandbox_types.go

```diff
@@ -66,6 +66,11 @@ type SandboxSpec struct {
 	// PodTemplate describes the pod spec that will be used to create an agent sandbox.
 	// +kubebuilder:validation:Required
 	PodTemplate PodTemplate `json:"podTemplate" protobuf:"bytes,3,opt,name=podTemplate"`
+
+	// ShutdownTime - Absolute time (RFC 3339) when the sandbox is deleted.
```

Comment:
Sandbox is deleted or Pod is deleted?

An object that deletes itself is going to be hard to use declaratively...

Reply:
Sandbox is deleted. Should we delete the pod instead and retain the sandbox with some status ?